README.md

```diff
@@ -77,7 +77,7 @@ Having problems or solved a problem? Check out the Islandora google groups for a
 
 Current maintainers:
 
-* [Daniel Aitken](https://github.com/qadan)
+* [Nat Kanthan]
```

Comment:
This is missing the github profile page. You can find Nat's profile link in the CLAW (islandora 8) repository

Reply:
Thanks Diego - have added the profile now.

Reply:
@DiegoPino can confirm that @cderven added the GH profile link. Looks like we're good to go to me.